README.md

```diff
@@ -1,67 +1,58 @@
-# AirbrakesV2
-
+# Air BrakesV2 🚀
 
 ## Overview
-This project controls the extension of airbrakes using a servo motor. It includes modules for logging data and managing the servo motor.
-
-### Installation
-
-#### Install uv:
-
-[uv](https://docs.astral.sh/uv/getting-started/installation/) is a python and project manager, which makes handling different python versions and virtual enviornments easier.
-
-`curl -LsSf https://astral.sh/uv/install.sh | sh`
-
-This project uses Python 3.12. Using an older version may not work since we use newer language features:
-
-`uv python install 3.12`
-
-#### Clone the repository:
-
-```
-git clone https://github.com/yourusername/AirbrakesV2.git
-cd AirbrakesV2
-```
-
-#### Set up a virtual environment:
-
-```
-uv venv
-source .venv/bin/activate
+This project is for controlling our Air brakes system with the goal of making our rocket "hit" its target apogee. We have a Raspberry Pi 4 as the brains of our system which runs our code. It connects to a servo motor to control the extension of our air brakes and an [IMU](https://www.microstrain.com/inertial-sensors/3dm-cx5-25) (basically an altimeter, accelerometer, and gyroscope). The code follows the [finite state machine](https://www.tutorialspoint.com/design_pattern/state_pattern.htm) design pattern, using the [`AirbrakesContext`](https://github.com/NCSU-High-Powered-Rocketry-Club/AirbrakesV2/blob/main/airbrakes/airbrakes.py) to manage interactions between the states, hardware, logging, and data processing. 
+
+https://github.com/user-attachments/assets/0c72a9eb-0b15-4fbf-9e62-f6a69e5fadaa
```

Comment:
nice, I thought this would be too sensitive / confidential information lol

Reply:
Ehh I don't love the idea of keeping secrets. If someone directly tells us to close things down, we will, but until then I like keeping it open.